calamari-server-trusty/config/definitions/calamari-server-trusty.yml

```diff
@@ -2,7 +2,10 @@
     block-downstream: false
     block-upstream: false
     builders:
-    - shell: !include-raw ../../build/build
+    - shell:
+      !include-raw:
```

Comment:
this needs to be outdented 2 spaces it seems. Like:

    - shell:
           !include-raw: